aocd/utils.py

```diff
@@ -1,16 +1,20 @@
+from __future__ import annotations
+
```

Comment:
To be honest, I suspect that the whole `utils` module should really be named `_utils`, and not be part of the public API at all.

But that's up to the maintainer of the project, really.